.gitignore

```diff
@@ -31,6 +31,7 @@ client/src/featureform/proto/
 .ipynb_checkpoints
 iris.csv
 provider/scripts/spark/tests/test_files/output
+**/grpc_debug.log
```

Comment:
i don't think we need this yet

Reply:
Awww, gotcha. Artifact of our other work. Will remove. 🙌